### PR TITLE
feat(explorer/ui): be table hookup and mobile styling

### PR DESCRIPTION
**Pull request by CaidynLuckyBeard**
was able to get the back end working again hooked up the table got some mobile updates.

task: https://app.asana.com/0/1206208509925075/1206970544933858

### CODE REVIEW

File: explorer/ui/app/components/details/From.tsx [hunk 1]
```diff
@@ -15,16 +16,22 @@ export const From = () => {
...
       </div>
       {/* Offset */}
       <div className="flex mt-5 pb-2 border-b-2 border-gray-500 border-solid">
         <p className="w-48">Tx Hash</p>
-        <Link to="/" className="underline text-indigo-400">
+        <Link to="/" className="underline text-indigo-400 hidden lg:block">
           0x109f40f806567158aaad05e43afe240cf394608cacd0016466dfb24dce2927d4
         </Link>
+        <Link to="/" className="underline text-indigo-400 block lg:hidden">
+          {hashShortener('0x109f40f806567158aaad05e43afe240cf394608cacd0016466dfb24dce2927d4')}
```
Comment (pavelnikolov) [inline explorer/ui/app/components/details/From.tsx:33]: why is this hash hard-coded?

File: explorer/ui/app/components/home/messageDataTable.tsx [hunk 2]
```diff
@@ -21,13 +21,14 @@ import { copyToClipboard } from '~/lib/utils'
...
 
 export default function XMsgDataTable() {
-
   const data = useLoaderData<XmsgResponse>()
   const revalidator = useRevalidator()
   const searchFieldRef = React.useRef<HTMLInputElement>(null)
 
   const pageLoaded = React.useRef<boolean>(false)
 
+  console.log(data)
```
Comment (pavelnikolov) [inline explorer/ui/app/components/home/messageDataTable.tsx:30]: Please, remove all console.log or debugging code

File: explorer/ui/app/components/home/messageDataTable.tsx [hunk 3]
```diff
@@ -46,15 +47,14 @@ export default function XMsgDataTable() {
...
-  const sourceChainList = data.supportedChains.map(chain => ({
-    value: chain.ChainID,
-    display: chain.DisplayName,
-    icon: chain.Icon,
-  }))
+  // const sourceChainList = data?.supportedChains.map(chain => ({
+  //   value: chain.ChainID,
+  //   display: chain.DisplayName,
+  //   icon: chain.Icon,
+  // }))
```
Comment (pavelnikolov) [inline explorer/ui/app/components/home/messageDataTable.tsx:54]: Remove commented out code

**Review by pavelnikolov — APPROVED** [review]

File: explorer/ui/app/components/details/From.tsx (hunk 1 above)
Comment (CaidynLuckyBeard) [reply explorer/ui/app/components/details/From.tsx:33]: this is still old code this will be fixed in the next PR

**Pull request merged**